wolfcrypt/test/test.c

```diff
@@ -22837,6 +22837,27 @@ static int myCryptoDevCb(int devIdArg, wc_CryptoInfo* info, void* ctx)
         info->rng.rng->devId = devIdArg;
     #endif
     }
+    else if (info->algo_type == WC_ALGO_TYPE_SEED) {
+    #ifndef WC_NO_RNG
+        static byte seed[] = { 0x00, 0x00, 0x00, 0x01 };
+        word32 len;
+        int i;
+
+        /* wc_GenerateSeed is a local symbol so we need to fake the entropy. */
+        while (info->seed.sz > 0) {
+            len = sizeof(seed);
+            if (info->seed.sz < len)
+                len = info->seed.sz;
+            XMEMCPY(info->seed.seed, seed, sizeof(seed));
+            info->seed.seed += len;
+            info->seed.sz -= len;
+            for (i = 0; i < (int)sizeof(seed) && ++seed[i] == 0; i++) {
```

Comment:
Can you add an inline comment here about skipping leader zeros?

Reply:
Changed: seed cast to word32 and incremented.